core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/groupBy.kt

```diff
@@ -9,27 +9,42 @@ class GroupByTests {
 
     @Test
     fun `groupBy values with nulls`() {
-        val df = dataFrameOf("a", "b")(
+        val df = dataFrameOf(
```

Comment:
i don't think this file should be reformatted in this commit / PR. It makes it harder to see actual changes. i mean, there are none, right?

Reply:
Unfortunately I have many of these reformats :/ Also for files I've touched but reverted in later commits. So there's no beginning trying to revert them all again I think. We could also view them as one of the "small fixes" part of the PR if that's okay with you :)

Reply:
Sure, let's keep these changes. Just saying for the future